mrp_flattened_bom_xlsx_labour_cost/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 ForgeFlow S.L. (http://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Export Flattened BOM to Excel with labour cost",
+    "version": "14.0.1.0.0",
+    "category": "Manufacturing",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/manufacture-reporting",
+    "license": "AGPL-3",
+    "depends": ["mrp_flattened_bom_xlsx_direct_materials_cost", "mrp_workorder"],
```

Comment:
@BernatPForgeFlow I just realized, you are depending on an enterprise module, is this dependency strictly needed?